DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -23,7 +27,7 @@
   <link rel="icon" href="@Url.Content("~/favicon.ico")">
 
   <!-- Open Graph -->
-  <meta property="og:url" content="@Context.Request.GetDisplayUrl()">
+  @*<meta property="og:url" content="@Context.Request.GetDisplayUrl()">*@

```

Comment:
Do we not need this? If so, probably worth removing instead of commenting out 🙂

Reply:
Unsure on this one, commented out because it was erroring. Is it important? - wasn't sure what it did tbh

Reply:
It's for defining how the link appears when sharing on social media (https://css-tricks.com/essential-meta-tags-social-media/) so fine to remove if you don't want these pages to be shared on social media 🙂